### PR TITLE
Second review

### PR DESCRIPTION
**Pull request by kaitcham**
For this brach, I followed these [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md).

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name ='Pikachu';
+SELECT name,escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
```
Comment (DeliceLydia) [inline queries.sql:14]: - [ ] Here you should select `species` you can do `SELECT species from animals;`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 1;
+COMMIT;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT Min(weight_kg) FROM animals;
+SELECT MAX(weight_kg) FROM animals;
+SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
```
Comment (DeliceLydia) [inline queries.sql:45]: - [ ] Please add a query where you are checking `Who escapes the most, neutered or not neutered animals?`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @kaitcham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Lordkaito — APPROVED** [review]
Hi @kaitcham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**